<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>COverlayMessages</class>
 <widget class="QFrame" name="COverlayMessages">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Overlay messages</string>
  </property>
  <layout class="QVBoxLayout" name="vl_StatusMessages">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QFrame" name="fr_StatusMessagesComponentsInner">
     <layout class="QVBoxLayout" name="vl_StatusMessagesComponentInner">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="fr_Header">
        <layout class="QHBoxLayout" name="hl_Header">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lbl_Header">
           <property name="text">
            <string>Status messages</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="hs_Header">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>309</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="tb_Close">
           <property name="text">
            <string>close</string>
           </property>
           <property name="icon">
            <iconset resource="../misc/misc.qrc">
             <normaloff>:/diagona/icons/diagona/icons/cross-button.png</normaloff>:/diagona/icons/diagona/icons/cross-button.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="sw_StatusMessagesComponent">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="pg_StatusMessages">
         <layout class="QVBoxLayout" name="vl_PgStatusMessages">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="swift::gui::views::CStatusMessageView" name="tvp_StatusMessages">
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pg_Image">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="lbl_Image">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>image will go here</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pg_StatusMessage">
         <layout class="QVBoxLayout" name="vl_PgStatusMessage">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="swift::gui::components::CStatusMessageForm" name="form_StatusMessage">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pg_StatusMessageSmall">
         <layout class="QVBoxLayout" name="vl_StatusMessageSmall">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="swift::gui::components::CStatusMessageFormSmall" name="form_StatusMessageSmall"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pg_TextMessage">
         <layout class="QVBoxLayout" name="vl_TextMessage">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QGridLayout" name="gl_TextMessage">
            <property name="spacing">
             <number>9</number>
            </property>
            <item row="1" column="3">
             <widget class="QLineEdit" name="le_TmTo">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="lbl_Receiver">
              <property name="text">
               <string>Received</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="lbl_TmTo">
              <property name="text">
               <string>to</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="lbl_TmFrom">
              <property name="text">
               <string>From</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="le_TmReceived"/>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="le_TmFrom">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="0" colspan="4">
             <widget class="QTextEdit" name="te_TmText"/>
            </item>
            <item row="0" column="0" colspan="4">
             <widget class="QWidget" name="wi_TmSupervisor" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QLabel" name="lbl_Supervisor">
                 <property name="text">
                  <string>This is a supervisor message</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pg_OverlayTextMessage">
         <layout class="QVBoxLayout" name="vl_OverlayTextMessage">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="swift::gui::components::CTextMessageComponent" name="comp_OverlayTextMessage">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pg_HTMLMessage">
         <layout class="QVBoxLayout" name="vl_HTMLMessage">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTextEdit" name="te_HTMLMessage"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pg_ProgressBar">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item alignment="Qt::AlignVCenter">
           <widget class="QFrame" name="fr_PRogressBar">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="vl_ProgressBar">
             <item>
              <widget class="QProgressBar" name="pb_ProgressBar">
               <property name="value">
                <number>10</number>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="lbl_ProgressBar">
               <property name="text">
                <string>text will go here</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="fr_Confirmation">
     <layout class="QHBoxLayout" name="hl_ConfirmationFrame">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QLabel" name="lbl_Confirmation">
        <property name="text">
         <string>Message goes here</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_Ok">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_Cancel">
        <property name="text">
         <string>Cancel</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="hs_Confirmation">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>swift::gui::views::CStatusMessageView</class>
   <extends>QTableView</extends>
   <header>gui/views/statusmessageview.h</header>
  </customwidget>
  <customwidget>
   <class>swift::gui::components::CStatusMessageForm</class>
   <extends>QFrame</extends>
   <header>gui/components/statusmessageform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::components::CStatusMessageFormSmall</class>
   <extends>QFrame</extends>
   <header>gui/components/statusmessageformsmall.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::components::CTextMessageComponent</class>
   <extends>QFrame</extends>
   <header>gui/components/textmessagecomponent.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pb_Ok</tabstop>
  <tabstop>pb_Cancel</tabstop>
  <tabstop>tb_Close</tabstop>
  <tabstop>tvp_StatusMessages</tabstop>
  <tabstop>le_TmReceived</tabstop>
  <tabstop>le_TmTo</tabstop>
  <tabstop>le_TmFrom</tabstop>
  <tabstop>te_TmText</tabstop>
 </tabstops>
 <resources>
  <include location="../misc/misc.qrc"/>
 </resources>
 <connections/>
</ui>
